src/Domain/LeanCode.DomainModels.MassTransitRelay/InjectLifetimeScopeFilter.cs

```diff
@@ -15,19 +13,19 @@ public void Probe(ProbeContext context)
 
         public Task Send(ConsumeContext<T> context, IPipe<ConsumeContext<T>> next)
         {
-            if (!context.TryGetPayload(out Autofac.Extensions.DependencyInjection.AutofacServiceProvider scopeProvider))
+            if (!context.TryGetPayload(out Autofac.Extensions.DependencyInjection.AutofacServiceProvider? scopeProvider))
```

Comment:
Now I wonder if this class is necessary too. We needed it to patch MT Autofac integration, but right now it's all Microsoft.Extensions